NEWS

```diff
@@ -1,3 +1,11 @@
+* Version 1.2.8 (unreleased)
+- Adding EAP support. radcli will now automatically add
```

Comment:
I just figured out this version bump. It is not necessary. This feature will be included in 1.2.7.